pkg/destroy/ovirt/destroyer.go

```diff
@@ -144,6 +148,25 @@ func (uninstaller *ClusterUninstaller) removeTemplate(con *ovirtsdk.Connection)
 	return nil
 }
 
+func (uninstaller *ClusterUninstaller) removeAffinityGroups(con *ovirtsdk.Connection) error {
+	cID := uninstaller.Metadata.Ovirt.ClusterID
+	affinityGroupService := con.SystemService().ClustersService().ClusterService(cID).AffinityGroupsService()
+	res, err := affinityGroupService.List().Send()
+	if err != nil {
+		return err
+	}
+	for _, ag := range res.MustGroups().Slice() {
+		if strings.Contains(ag.MustName(), uninstaller.Metadata.InfraID) {
```

Comment:
Don't we only want to remove the affinity group if it has the prefix of "{INFRA_ID}-"?
```suggestion
		if strings.HasPrefix(ag.MustName(), fmt.Sprintf("%s-", uninstaller.Metadata.InfraID)) {
```

Reply:
Well, it is a unique string so contains felt safe enough but sure.
Done